### ISSUE TITLE
is it possible to paralyze the Guided Local Search algorithm?

### ISSUE THREAD

**Issue by samilAyoub** [google/or-tools#3088]
**What version of OR-Tools and what language are you using?**
Version: master/v7.3/v7.2 etc.
Language: C++/Java/Python/C#
python
**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
Guided Local Search
**What operating system (Linux, Windows, ...) and version?**
Windows
**What did you do?**
I'm using the Guided Local Search algorithm, but it takes a too long to find a solution, I wonder if there is a way to paralyze it?

**Comment by Mizux** [comment]
GLS from routing/constraint_solver solver ?

**Comment by samilAyoub** [comment]
@Mizux yes

**Comment by lperron** [comment]
no, unless you problem has a lot of side constraints that makes it mostly infeasible.
